cdap-standalone/src/main/resources/logback-container.xml

```diff
@@ -17,7 +17,7 @@
   -->
 
 <!--
-  Logback used by containers that run in YARN, such as Explore, MapReduce, Spark
+  Logback used by containers that run in YARN, such as  MapReduce, Spark
```

Comment:
nit: extra whitespace in `as  MapReduce`

Reply:
Resolved